internal/servers/worker/worker.go

```diff
@@ -28,6 +28,7 @@ type Worker struct {
 	started     *ua.Bool
 
 	controllerStatusConn *atomic.Value
+	workerStartTime      *atomic.Value
```

Comment:
This doesn't need to be an atomic value since it will never change -- it will be set on startup and only read thereafter. It can be a plain `time.Time`.

Reply:
Fixed!